### ISSUE TITLE
[DetailList] Screen reader doesn't announce searched results at filter edit fields.

### ISSUE THREAD

**Issue by zhr1130** [microsoft/fluentui#18293]
### Environment Information

- **Package version(s)**: 8.14.13
- **Browser and OS versions**: Windows 10 with Narrator

### Describe the issue:

Narrator doesn't announce searched results at filter edit fields while user try with valid and invalid details on DetailList and due to this, user is unable to get an idea about searched results.

### Please provide a reproduction of the issue in a codepen:

https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist/basic

#### Actual behavior:

Screen reader doesn't announce searched results at filter edit fields while user try with valid and invalid details on DetailList and due to this, user is unable to get an idea about searched results.

#### Expected behavior:

Screen reader should announce searched results at filter edit fields while user try with valid and invalid details on DetailList so that screen reader user could get an idea of searched results.

### Documentation describing expected behavior

MAS Reference:
MAS 1.3.1 - Info and Relationships

**Comment by gouttierre** [comment]
@zhr1130 - thanks for filing this issue with us.

@miroslavstastny - Would you be able to confirm if this is a regression, or if this behavior is an issue?
cc @smhigley

**Comment by smhigley** [comment]
This is an issue with our example implementation rather than DetailsList itself, since the filter input isn't built into the component.

To do this right, we'd need to set up a timeout to debounce updates while typing, and only fire a live region with the number of updated results after about a half second of inactivity. I'll defer to @miroslavstastny on whether we should implement that in this example page, since it's not directly a part of the DetailsList.
